.travis.yml

```diff
@@ -2,15 +2,20 @@
 # This file will be regenerated if you run travis_pypi_setup.py
```

Comment:
I wonder if we should remove this and instead use github actions similar to https://github.com/release-depot/flask-container-scaffold/blob/main/.github/workflows/test.yml

Reply:
Agree, travis has not been great recently, and I think actions are now the preferred method

Reply:
I agree with the idea as well. I tried to update this PR with a first attempt at a github action, though I wouldn't be surprised if it takes a couple iterations to get it working!

Reply:
I think what you have looks fine, we can always do a rev to fix CI if needed.  Are you wanting to leave travis file in until we see if the new actions work correctly? If so, we could easily test the action with the next patch removing travis (the other alternative would be to temporarily not limit push action tests to be on main (well, I guess this is still using 'master'), and once you see the action work on your own branch, you will know you can limit it to just main for the patch).

Reply:
Sounds reasonable to me! And yes, the repo was created a couple of years ago so it still uses master.

Reply:
ok - maybe as a next step before adding more working code, we should move this to the newer 'main' standard?  We need to start shifting everything anyway.  I am approving this, looks good!